### PR TITLE
feat: kiosk_mode update [do not squash]

### PR DESCRIPTION
**Pull request by ookami-kb**

### CODE REVIEW

File: kiosk_mode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.2.0
+
+> Note: This release has breaking changes.
+
+ - **FEAT**: Add `isManagedKiosk` method (#206).
+ - **FEAT**: Track Kiosk mode changes on Android (#205).
+ - **BREAKING** **FEAT**: Update readme.
```
Comment (Sevastyan) [inline kiosk_mode/CHANGELOG.md:7]: How is a README update a breaking change? Shouldn't it be for #206?

**Review by Sevastyan — APPROVED** [review]
Otherwise, LGTM.

File: kiosk_mode/CHANGELOG.md (hunk 1 above)
Comment (ookami-kb) [reply kiosk_mode/CHANGELOG.md:7]: It should, but you didn't put it there and I missed it during review 😄

Comment (Sevastyan) [reply kiosk_mode/CHANGELOG.md:7]: But I did put it there 🤔
Screenshot_2022-05-20_17-15-46

Comment (ookami-kb) [reply kiosk_mode/CHANGELOG.md:7]: It's squashed, so the description isn't going into the final commit. It's better to use "!" in the commit message.

**Pull request merged**